src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from 'components/App'
+import './index.css'
+
+ReactDOM.render(
+  <App />,
+  document.getElementById('root')
+);
```

Comment:
Maybe try to get hot reloading working?

https://medium.com/@sheepsteak/adding-hot-module-reloading-to-create-react-app-e053fadf569d